core/api/views/cp_reports.py

```diff
@@ -387,6 +388,9 @@ def put(self, request, *args, **kwargs):
 
         current_obj.delete()
 
+        if new_instance.status == CPReport.CPReportStatus.FINAL:
+            send_mail_report_submit.delay(new_instance.id)  # send mail to MLFS
```

Comment:
We should also test that tasks sending emails get called when needed (use `CELERY_ALWAYS_EAGER` or something similar)